examples/internal/proto/examplepb/echo_service.swagger.json

```diff
@@ -621,9 +621,7 @@
             "name": "no",
             "in": "body",
             "required": true,
-            "schema": {
-              "$ref": "#/definitions/examplepbEmbedded"
-            }
+            "schema": {}
```

Comment:
This change looks wrong, no? It seems we're losing some definitions here.

Reply:
yes indeed. I think I messed up the generation. I cleaned everything and re-generated the protos and the changes are gone.